### ISSUE TITLE
Proposal: new mutopia header field to track typesetter and maintainer separately

### ISSUE THREAD

**Issue by dominicus** [MutopiaProject/MutopiaProject#405]
We have several pieces where the original typesetter no longer provides maintenance, or contact email is no longer valid.

Propose defining a new mutopia header to track current maintainer separate from original typesetter (also copyright holder for CC-licensed pieces).

Example:
\header{
...
  typesetter = "Joe Original"  
  typesetterEmail = "joe (at) mysite.com
  maintainer = "John New"
  maintainerEmail = "john (at) newsite.com"
...
}
- typesetter: mandatory...needed to generate copyright attribution on PDF's.
- typesetterEmail: optional
- maintainer: optional, defaults to typesetter if undefined.
- maintainerEmail: required when maintainer is defined and different from typesetter, otherwise leave undefined.
- udpate copyright attribution code to refer to \typesetter (instead of current \maintainer field)
- website lists 'typesetter'
- if defined, website lists maintainer
- Website lists the first one of these that happens to be defined: maintainerEmail, typesetterEmail, contributions@mutopia.com.

Would need some ground rules on minimum due-diligence to confirm original typesetter is no longer willing or available to maintain the piece before a different maintainer is defined.

**Comment by dominicus** [comment]
Perhaps it's best to focus on the problem areas separately, and along the lines we can make progress on.

We seem to agree re-branding the current `\maintainer` header to `\typesetter` will clarify what this field is tracking long-term: the name of the initial typesetter, to which copyright (or PD dedication) belongs, and used for the  `\copyright` attribution statement.

A related, but separate issue is how to facilitate the reporting of errors that users may find while reviewing a piece, and how to best get this report routed to a contributor willing to validate and implement the correction (i.e. a maintainer).

I hear implementing the  `\maintainer` -> `\typesetter` switch would be possible to do for future contributions and update (i.e. similar to the recent change to `\license`) without adding significant complexity or requiring a mass-change.  True?

The issue of identifying current maintainer, I can peel to a separate issue....and I agree: maintenance assignments can be a fluid thing in the lifecycle of a piece, and there's no requirement or convenience in tracking this via LilyPond header.

**Comment by chrissawer** [comment]
> I hear implementing the \maintainer -> \typesetter switch would be possible to do for future contributions and update (i.e. similar to the recent change to \license) without adding significant complexity or requiring a mass-change. True?

Yes, I think we could implement this in a similar way to how we changed to \license.
